packages/salesforcedx-vscode-lwc-next/src/index.ts

```diff
@@ -136,7 +136,9 @@ function startLWCLanguageServer(
         ),
         vscode.workspace.createFileSystemWatcher(
           '**/lightningcomponents/*/*.js'
-        )
+        ),
+        // need to watch for directory deletions as no events are created for contents or deleted directories
+        vscode.workspace.createFileSystemWatcher('**/', true, true, false)
```

Comment:
@rsalvador - Copied what you had in the internal vscode extension. This was the only change I noticed.